addons/project/static/tests/tours/project_sharing_tour.js

```diff
@@ -88,6 +88,7 @@ const projectSharingSteps = [...stepUtils.goToAppSteps("project.menu_main_pm", '
 }, {
     trigger: 'iframe button.o_switch_view.o_list',
     content: 'Go to the list view',
+    run: 'click',
```

Comment:
remove this
You can either:
- Remove this line and add a new step afterward which correctly checks that the "Go to list view" correctly behaved: just a check step about list view being loaded
- Or simply remove this step, it's probably doing nothing: it clicks and directly stops the test without waiting anything (which lead to race condition generally, because it stops the second it clicks on the button, then the tour manager is killing everything but there is still pending request which can't be handled correctly).